### PR TITLE
Adds a medical scan verb for ghosts

### PR DESCRIPTION
**Pull request by Jaraci**
Allows ghosts to health analyse whatever target they're following. I ported this from Baystation and wiggled it a bit to get it to work on Aurora. Very cool!

image

image

### CODE REVIEW

**Comment by Jaraci** [comment]
!review

File: code/modules/mob/abstract/observer/observer.dm [hunk 1]
```diff
@@ -311,6 +311,16 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
...
 
+/mob/abstract/observer/verb/scan_target()
+	set category = "Ghost"
+	set name = "Medical Scan Target"
+	set desc = "Analyse the health of whatever you are following."
+
+	if(ishuman(following))
+		health_scan_mob(following, usr, TRUE, TRUE)
+
+	else to_chat(src, "<span class='notice'>Not a scannable target.</span>")
```
Comment (Geevies) [inline code/modules/mob/abstract/observer/observer.dm:322]: please make this indented under the else. also, make it give different messages. for instance, if i'm following a non-human, it gives me the current one, if i'm not following anyone, it says something like "Not following a target!"

Comment (Geevies) [reply code/modules/mob/abstract/observer/observer.dm:322]: also replace the "<span class='notice'>Not a scannable target.</span>" stuff with SPAN_WARNING("Not a scannable target.")

**Review by Geevies — CHANGES_REQUESTED** [review]

File: code/modules/mob/abstract/observer/observer.dm (hunk 1 above)
Comment (Jaraci) [reply code/modules/mob/abstract/observer/observer.dm]: I implemented these, I think!

**Review by Geevies — APPROVED** [review]
looks great, good work

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**